samples/bluetooth/fast_pair/locator_tag/configuration/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Align the TX power encoded in the Fast Pair advertising set and
+# the Read Beacon Parameters response with Fast Pair expectations.
+# The value has been tailored for following DKs:
+#   * nrf54l15dk/nrf54l15/cpuapp
+CONFIG_BT_ADV_PROV_TX_POWER_CORRECTION_VAL=-17
```

Comment:
btw. Do we want to double check the value for the nRF54L15 DK?

Reply:
same